src/project/ProjectNameEditable.tsx

```diff
@@ -74,7 +49,7 @@ const ProjectNameEditable = ({
         fontSize="xl"
         color="brand.500"
         variant="ghost"
-        onClick={handleEdit}
+        onClick={() => actions.editProjectName()}
```

Comment:
You can pass actions.editProjectName directly (and avoid recreating a function each render). That's why they're arrow functions in ProjectActions.

Reply:
Typescript complains because editProjectName has an optional param. I've rejigged it [`6b7b162`](https://github.com/microbit-foundation/python-editor-next/pull/763/commits/6b7b162c9c7399d766a75310e87e258351a08f55).